index.rst

```diff
@@ -110,9 +110,9 @@ else than just writing your algorithm!
   Check out `Writing Bears`_ for more
   information on this.
 
-To programmatically access coala's functionality, use ``coala-json`` or
-``coala-format`` if you want to use a custom format string. Both of those
-binaries, among with ``coala-ci``, are noninteractive and thus suitable for
+To programmatically access coala's functionality, use ``coala --json`` or
+``coala --format`` if you want to use a custom format string. Both of those
+binaries, among with ``coala --ci``, are noninteractive and thus suitable for
```

Comment:
I thought i have to rephrase in FAQ.rst only! Sorry for this!
I will make changes